src/modules/services/submodule/addressbook/components/EditContactModal.vue

```diff
@@ -17,8 +17,8 @@
               <fieldset class="w-full">
                 <div class="error error_box mb-5" v-if="err!=''">{{ err }}</div>
                 <div class="mb-5 text-center"><span class="text-lg text-gray-700">{{$t('addressbook.editcontact')}}</span></div>
-                <TextInput placeholder="Name" errorMessage="Name required" v-model="contactName" icon="id-card-alt" :showError="showNameErr" />
-                <TextInput placeholder="Address" :errorMessage="addErr" v-model="address" icon="wallet" :showError="showAddErr" />
+                <TextInput :placeholder="$t('services.name')" :errorMessage="$t('services.namevalidation')" v-model="contactName" icon="id-card-alt" :showError="showNameErr" />
+                <TextInput :placeholder="$t('createsuccessful.address')" :errorMessage="addErr" v-model="address" icon="wallet" :showError="showAddErr" />
```

Comment:
maybe use same module for the translate message ?

Reply:
but the value is the same, maybe i create a general module to avoid unnecessary translations

Reply:
for instance, general.address